modules/tinymce/src/themes/silver/test/ts/browser/editor/EventsTest.ts

```diff
@@ -0,0 +1,73 @@
+import { Mouse, UiFinder, Waiter } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyContentActions, TinyDom, TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.themes.silver.editor.EventsTest', () => {
+  const settings = {
+    base_url: '/project/tinymce/js/tinymce',
+    plugins: 'link',
+    menubar: 'file'
+  };
+
+  const hook1 = TinyHooks.bddSetupLight(settings, [ Theme ]);
+  const hook2 = TinyHooks.bddSetupLight(settings, [ Theme ]);
+  const menuSelector = '[role="menu"]';
+
+  const pWaitMenuToOpen = () =>
+    Waiter.pTryUntil('Wait for a menu to open', () => UiFinder.exists(SugarBody.body(), menuSelector));
+
+  const pWaitMenuToClose = () =>
+    Waiter.pTryUntil('Wait for all menus to close', () => UiFinder.notExists(SugarBody.body(), menuSelector));
+
+  it('TINY-7399: Clicking on the editor should close the menu', async () => {
+    const editor = hook1.editor();
+    TinyUiActions.clickOnMenu(editor, '[role="menuitem"]');
```

Comment:
Could be worth making `'[role="menuitem"]'` a constant like `menuSelector`.

Reply:
Created the constant, thanks for spotting that 👍